static/scss/layout.scss

```diff
@@ -165,19 +165,39 @@ body {
       border: 1px solid #dddddd;
       border-radius: 5px;
       background-color: white;
-      padding: 25px;
-
-      .card-image {
+      .card-user {
+        background-color: darkslategrey;
         @include span-columns(12);
-        padding: 25px 0px;
+        .card-image-box {
+          @include span-columns(2);
+          min-width: 150px;
+          height: 220px;
+          padding-top: 40px;
+          .card-image{
+            border-radius: 50%;
+            height: 140px;
+            width: 140px;
+            float: right;
```

Comment:
Can you use `margin-left` here instead to position the image? I'd like to avoid `float` as much as possible